backend/app/crud/users_table.py

```diff
@@ -15,6 +15,15 @@ def __init__(self, db: Database):
         self.table_id = optional_table_id
         self.db = db
 
+    def parse_user_data(self, user: dict) -> dict:
+      user_data = user.get('fields', {})
+      user_data['id'] = user.get('id', '')
```

Comment:
I think this part of the code should be in the AirtableDB class, as it will be used for every crud we will use. What do you think?